workflows/in_memory_index.md

```diff
@@ -0,0 +1,55 @@
+**Usage for in-memory indices**
+================================
+
+To generate index, use the `tests/build_memory_index` program. 
+--------------------------------------------------------------
+
+```
+./tests/build_memory_index  data_type<int8/uint8/float>  dist_fn<l2/mips>   data_file.bin  output_index_file  R(graph degree)   L(build complexity)  alpha(graph diameter control)   T(num_threads)
+```
+
+The arguments are as follows:
+
+(i) **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported. 
```

Comment:
If you use numbers like:

```
1. **data_type**: ...
2. **dist_fn**: ...
```
You'll get formatting like:

1. **data_type**: The type of dataset you wish to build an index. float(32 bit), signed int8 and unsigned uint8 are supported.
2. **dist_fn**: There are two distance functions supported: minimum Euclidean distance (l2)) and maximum inner product (mips).

I mean, if you want to get all fancified :)